ccd-definition/ComplexTypes.json

```diff
@@ -1 +1,155 @@
-[]
+[
+  {
+    "ID": "SolicitorReferences",
+    "ListElementCode": "solicitorReference",
+    "FieldType": "Text",
+    "ElementLabel": "Claimant's legal representative's reference",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "SolicitorReferences",
+    "ListElementCode": "defendantReference",
+    "FieldType": "Text",
+    "ElementLabel": "Defendant's legal representative's reference",
+    "SecurityClassification": "Public"
```

Comment:
The Ids for these attributes aren't clear. If I look at the labels, it looks like it should be claimantSolicitorReference and defendantSolicitorReference

Reply:
PR raised here https://github.com/hmcts/ucmc-service/pull/41 I have changed to claimantReference. The full data will read solicitorReferences.claimantReference etc